README.md

```diff
@@ -242,12 +242,18 @@ Here are some examples of how to configure it:
 ```
 
 Pagination properties:
-  - Scheme -- if specified, must be 'query'
-  - PageNumberKey -- query string key for the page number
-  - PageSizeKey -- number of items per page, default is 100
-  - FirstPage -- number of the first page, default is 1
+  - Scheme -- if specified, must be "pages" for page based or "items" for item based
+  - FirstItemIndex -- (item based) index of first item to fetch, default is 0
```

Comment:
Maybe use two separate unordered lists, one with a "pages scheme:" header and another with an "item scheme:" header instead of using "(item based)" and "(page based)" in the description.